tests/unit/queries/attribute-test.js

```diff
@@ -55,3 +55,11 @@ it('uses page scope', function(assert) {
 
   assert.equal(attr(), 'Logo small');
 });
+
+it('searches for element by index if provided', function(assert) {
+  fixture('<img alt="img1" class="img"></img><img alt="img2" class="img"></img>');
```

Comment:
`<img>` tags doesn't support child elements. Would you mind changing the fixture tu use `<span>` elements instead?